components/Footer.js

```diff
@@ -11,6 +11,8 @@ import Button from "./Button";
 import Link from "@components/Link";
 
 export default function Footer() {
+  const isExternalUrl = (url) => /^(?:[a-z+]+:)?\/\//i.test(url)
```

Comment:
It would be more readable if we add the regex in a const variable.

const EXTERNAL_URL_REGEX = /^(https?|ftp):\/\//i;

const isExternalLink = (url, regex = EXTERNAL_URL_REGEX) => regex.test(url);

Something like this, we can try.

Reply:
Good suggestion, I'll try to improve readability.